examples/29-update-nodegroup.yaml

```diff
@@ -0,0 +1,10 @@
+---
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-29
+  region: us-west-2
+
+managedNodeGroups:
+- name: mng
```

Comment:
what's this example config demonstrating?

Reply:
nothing right now, this PR is just for bootstrapping `eksctl update nodegroup`! I'm creating another PR to use `update nodegroup` to update `updateConfig!

Reply:
see: https://github.com/weaveworks/eksctl/pull/3855/commits/2f353953cddf57eb99fdc641a4941aacc2e7279b